mon-pix/app/components/module/component/_stepper.scss

```
@@ -70,6 +70,11 @@
     border: 1px solid var(--pix-neutral-100);
     border-radius: 8px;
     transform: translateX(calc(-1 * var(--stepper-gap-between-steps-with-step-width) * var(--current-step-index)));
+    transition: transform .8s ease;
```

Penser à respecter les préférence de `reduce-motion`

---

Je pense qu'on peut accélerer un peu 
```suggestion
    transition: transform .5s ease;
```

---

J'ai l'impression que `ease-in-out` est moins vénère